### PR TITLE
Add SURVIVABILITY_MODE_DETECTED Event type to Sysman API

### PR DESCRIPTION
**Pull request by sanilkumar0**
Add 'SURVIVABILITY_MODE_DETECTED' event

### CODE REVIEW

File: scripts/sysman/events.yml [hunk 1]
```diff
@@ -44,6 +44,8 @@ etors:
       desc: "Event is triggered when accelerator RAS uncorrectable errors cross thresholds (use $sRasSetConfig() to configure - disabled by default)."
     - name: DEVICE_RESET_REQUIRED
       desc: "Event is triggered when the device needs to be reset (use $sDeviceGetState() to determine the reasons for the reset)."
+    - name: SURVIVABILITY_MODE_DETECTED
+      desc: "Event is triggered when graphics driver encounter error condition and Yellow Bang (in Window)."
```
Comment (wdamon-intel) [inline scripts/sysman/events.yml:48]: After the `desc:` add versioning.

`version: 1.9`

Also, the Spec is platform agnostic. Remove platform-specific language, or rephrase to not exclude any particular platform(s).

i.e. `Event is triggered when the driver encounters an error condition.`

**Comment by wdamon-intel** [comment]
What issue, if any, does this resolve?

**Comment by sanilkumar0** [comment]
Rephrase the description and added version tag.

**Comment by sanilkumar0** [comment]
This feature is to notify as an event, if the graphics driver stops working

**Comment by aravindksg** [comment]
> What issue, if any, does this resolve?

Added issue for this here: https://github.com/oneapi-src/level-zero-spec/issues/253

@sanilkumar0  could you please update commit message to say "Resolves: issue number above"

File: scripts/sysman/events.yml [hunk 2]
```diff
@@ -44,6 +44,10 @@ etors:
       desc: "Event is triggered when accelerator RAS uncorrectable errors cross thresholds (use $sRasSetConfig() to configure - disabled by default)."
     - name: DEVICE_RESET_REQUIRED
       desc: "Event is triggered when the device needs to be reset (use $sDeviceGetState() to determine the reasons for the reset)."
+    - name: SURVIVABILITY_MODE_DETECTED
+      desc: 
+            "1.9": "Event is triggered when graphics driver encounter an error condition."
```
Comment (aravindksg) [inline scripts/sysman/events.yml:49]: format needs to be:
```
name: SURVIVABILITY_MODE_DETECTED
desc: "Event is triggered when graphics driver encounter an error condition."
version: "1.9"
```

For example: check temperature.yml file for "GPU_BOARD" enum

**Review by wdamon-intel — APPROVED** [review]

**Pull request merged**